spec/acceptance/clone_repo_spec.rb

```diff
@@ -479,7 +479,7 @@
       run_shell("mkdir -p #{homedir}/.ssh")
       run_shell("ssh-keygen -q -t rsa -f #{homedir}/.ssh/id_rsa -N ''")
 
-      run_shell('ssh-keygen -R localhost', expect_failures: true)
+      run_shell('rm ${homedir}/.ssh/known_hosts', expect_failures: true)
       run_shell("ssh-keyscan localhost >> #{homedir}/.ssh/known_hosts")
```

Comment:
Firstly - could avoid the `rm` above by just blatting the file with `>` instead of `>>`.

Secondly, `#{homedir}` here - will that be the homedir of the "target" user whose repo we're trying to check out, or that of the user running the Puppet code (root?)?  It wants to be the second, not the first.